npp.py

```diff
@@ -15,23 +15,41 @@
 ## ------- Set up our list of numbers -------
 S = [25, 7, 13, 31, 42, 17, 21, 10]
 
+# TODO: Enter your token here
+token = 'Your-Token-Here'
+
 ## ------- Set up our QUBO dictionary -------
 
 # TODO:  Add code here to define your QUBO dictionary
+def define_QUBO(S):
```

Comment:
Based on context, I think this function is meant to grab a number-partitioning QUBO for a set `S`. Hence, the function isn't allowing the end-user to define a QUBO, it's grabbing a very specific QUBO for the end-user.

Perhaps it can be written as
```
# TODO: Add code here ...
def get_qubo(S):
    ```Returns a number partitioning QUBO for a list of numbers S```

    Q = {}
    ..
```